bower.json

```diff
@@ -11,6 +11,7 @@
     "o-colors": "^5.0.0",
     "o-icons": "^6.0.0",
     "o-typography": "^6.0.0",
+    "o-editorial-typography": "^1.0.7",
```

Comment:
Is this now a major version bump because of this dependency addition?

Reply:
I don't think we should consider this major as there is only one version, so no conflicts.

Reply:
Excellent